Scripts/i18n.ps1

```
@@ -0,0 +1,591 @@
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+$RepoRoot = Split-Path -Parent $PSScriptRoot
+$I18nDir = Join-Path $RepoRoot '智绘教/src/i18n'
+$HeaderPath = Join-Path $RepoRoot '智绘教/IdtI18nKeys.g.h'
+$BaseLanguage = 'zh-CN'
+$BasePath = Join-Path $I18nDir "$BaseLanguage.jsonc"
+
+function Remove-JsoncComments {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Remove-JsoncComments' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)